rkovalenko/3Sum.cs

```diff
@@ -0,0 +1,44 @@
+public class Solution {
+    public IList<IList<int>> ThreeSum(int[] nums) {
+        Array.Sort(nums);
+        var hashSet = new HashSet<(int,int,int)>();
+
+        for(int i = 0; i < nums.Length - 1; ++i)
+        {
+            var j = i + 1;
+            var k = nums.Length - 1;
+
+            while (k < nums.Length)
+            {
+                if(k <= j || k <= i)
```

Comment:
nit: Maybe explain loop conditions and premature exit to make it obvious for readers next time. (I remember you asked what could have been commented)